addons/sale/views/sale_views.xml

```diff
@@ -688,6 +688,7 @@
             <field name="inherit_id" ref="sale.view_sales_order_filter"/>
             <field name="arch" type="xml">
                 <xpath expr="//filter[@name='my_sale_orders_filter']" position="replace">
+                    <field name="campaign_id"/>
```

Comment:
In first commit you add a field in sale views: this should probably be moved to utm / sale pimp commit :) .